src/builder.js

```diff
@@ -269,6 +273,13 @@ const FormBuilder = ({
       if (data[key] instanceof Date) {
         data[key] = data[key].toISOString()
       }
+      if (data[key] instanceof FileList) {
+        const reader = new FileReader()
+        reader.onload = function (e) {
```

Comment:
should we include here an  reader.onerror callback?

Reply:
Good spot! I have also realised that this should be asynchronous. Uploading fix!